web/packages/teleterm/src/ui/DocumentTerminal/Terminal/ctrl.ts

```diff
@@ -105,7 +105,6 @@ export default class TtyTerminal {
   }
 
   destroy(): void {
-    this.ptyProcess?.dispose();
```

Comment:
Removed this `dispose` from here. `ctrl.destroy()` is called from the cleanup function in the `useEffect` in `Terminal`. `useDocumentTerminal` already calls `ptyProcess.dispose` on unmounting.

What's more, `useDocumentTerminal` is the place which creates the ptyProcess, so it feels that it should also be responsible for disposing it. Similar to how in Go, the goroutine that owns a channel should also be responsible for closing the channel.